### PR TITLE
Added screenshots in assets folder and Updated README

### PR DESCRIPTION
**Pull request by Balram02**
Brand: MOTO G5 PLUS (Custom ROM)
Version: Oreo (8.1.0)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,17 @@ _Follow these instructions to build and run the project.._
 * Add any number of questions
 * Export all feedback to excel
 
+## Screenshots
+<div>
+   <img src="assets/Welcome_Screen.png" width="250"/>
+   <img src="assets/Add_question.png" width="250" hspace="20"/>
+   <img src="assets/User_info.png" width="250"/>
+   <img src="assets/Question_Screen_1.png" width="250" hspace="20"/>
+   <img src="assets/Question_Screen_2.png" width="250" />
```
Comment (arshadkazmi42) [inline README.md]: You can remove this.

File: README.md [hunk 2]
```diff
@@ -28,6 +28,17 @@ _Follow these instructions to build and run the project.._
...
+   <img src="assets/Thank_Message_Screen.png" width="250" hspace="20"/>
```
Comment (arshadkazmi42) [inline README.md]: You can remove this.

File: README.md [hunk 3]
```diff
@@ -28,6 +28,17 @@ _Follow these instructions to build and run the project.._
...
+   <img src="assets/Clear_feedback.png" width="250" hspace="20"/>
```
Comment (arshadkazmi42) [inline README.md]: You can remove this.

File: README.md [hunk 4]
```diff
@@ -28,6 +28,17 @@ _Follow these instructions to build and run the project.._
 * Add any number of questions
 * Export all feedback to excel
 
+## Screenshots
+<div>
```
Comment (arshadkazmi42) [inline README.md]: add `align="center"`

File: README.md [hunk 5]
```diff
@@ -28,6 +28,17 @@ _Follow these instructions to build and run the project.._
...
+   <img src="assets/Welcome_Screen.png" width="250"/>
+   <img src="assets/Add_question.png" width="250" hspace="20"/>
+   <img src="assets/User_info.png" width="250"/>
```
Comment (arshadkazmi42) [inline README.md]: You can remove this.

**Comment by arshadkazmi42** [comment]
I think 3 images are fine and moving all snapshots to the center will be better. You can do those changes and this PR will be ready to merge.
Also, tag the issue number in the PR comment `Closes #9`

**Comment by Balram02** [comment]
changed README.
The issue number is #11

**Comment by arshadkazmi42** [comment]
@Balram02 Looks great. thank you for working on this.

**Pull request merged**